src/containers/Experiment/ProcessingInformationCell.js

```diff
@@ -4,319 +4,259 @@ import ModalManager from '../../components/Modal/ModalManager';
       )}
       modalProps={{ className: 'processing-info-modal' }}
     >
       {() => (
-        <div>
-          <h1 className="processing-info-modal__title">
-            Processing Information
-          </h1>
-          <div className="dot-label dot-label--submitter">
-            Submitter processed
+        <ProcessingInformationModalContent
+          sample={sample}
+          results={computationalResults}
+        />
+      )}
+    </ModalManager>
+  );
+}
+
+class ProcessingInformationModalContent extends React.Component {
+  render() {
+    const { results, sample } = this.props;
+
+    const pipelinesText = results.map(result => result.processor.name);
+
+    return (
+      <div>
+        <h1 className="processing-info-modal__title">Processing Information</h1>
+        <div className="dot-label">refine.bio processed</div>
+
+        <h3>{stringEnumerate(pipelinesText)}</h3>
+
+        <div className="pipeline">
+          <div className="pipeline__item">
+            <img src={FileIcon} alt="" />
+            <div>Input File</div>
+          </div>
+          <div className="pipeline__arrow">
+            <div className="arrow" />
           </div>
 
-          <SubmitterSuppliedProtocol />
+          {results.map(({ processor: { name } }, index) => (
+            <React.Fragment key={index}>
+              <div className="pipeline__item">
+                <img src={ProcessIcon} alt="" />
+                <div>{name}</div>
+              </div>
+              <div className="pipeline__arrow">
+                <div className="arrow" />
+              </div>
+            </React.Fragment>
+          ))}
 
-          <section className="processing-info-modal__section">
-            <GeneIdentifierConversion />
-          </section>
+          <div className="pipeline__item">
+            <img src={FileIcon} alt="" />
+            <div>Gene Expression Matrix</div>
+          </div>
         </div>
-      )}
-    </ModalManager>
-  );
+
+        {results.map(({ processor: { name } }) =>
+          this._getProtocolDescription(name)
+        )}
+
+        <h3 className="processing-info-modal__subtitle">Version Information</h3>
+        <table>
+          <tbody>
+            {results.map(({ processor: { name, version } }) => (
+              <tr>
+                <td className="processing-info-modal__version">{name}</td>
+                <td>{version}</td>
+              </tr>
+            ))}
+
+            <GenomeBuild organism={sample.organism.name} />
+          </tbody>
+        </table>
+
+        <section className="processing-info-modal__section">
+          <SubmitterSuppliedProtocol {...this.props} />
+        </section>
+      </div>
+    );
+  }
+
+  static PROTOCOLS = {
+    'Affymetrix SCAN': AffymetrixScanProtocol,
+    'Illumina SCAN': IlluminaScanProtocol,
+    'Transcriptome Index': null,
+    'Salmon Quant': SalmonProtocol,
+    MultiQC: null,
```

Comment:
Let's enclose `MultiQC` and `Tximport` with ' ' too to keep the format consistent.

Reply:
I did it initially, but prettify changed it. By default object keys with a single word don't have `''`.

We could try to change the configuration of prettify, but it might not be worth it, since we have been using it everywhere till now.

Reply:
Never mind then. Sometimes Prettify is too smart :(

Reply:
I agree, I don't particularly like how they organize the html either, but I have gotten more used to it